airbyte-webapp/src/packages/cloud/views/auth/components/GitBlock/GitBlock.tsx

```diff
@@ -0,0 +1,34 @@
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { FC } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { useConfig } from "../../../../../../config";
+import styles from "./GitBlock.module.scss";
+
+export interface GitBlockProps {
+  title?: string;
+  titleStyle?: React.CSSProperties;
+  message?: string;
+  messageStyle?: React.CSSProperties;
+}
+export const GitBlock: FC<GitBlockProps> = ({ title, titleStyle, message, messageStyle }) => {
+  const config = useConfig();
+  return (
+    <div className={styles.container}>
+      <a className={styles.link} href={config.links.gitLink} target="_blank" rel="noreferrer">
+        <div className={styles.content}>
+          <FontAwesomeIcon icon={faGithub} className={styles.icon} />
+          <div>
+            <p className={styles.hostingText} style={titleStyle}>
+              {title ? title : <FormattedMessage id="login.selfhosting" />}
```

Comment:
Since we can overwrite all strings via LaunchDarkly, we won't need to offer this as properties on this components, since they can be overwritten anyway.